docs/config/MathJax.js

```diff
@@ -16,7 +16,7 @@ window.MathJax = {
 
 (function () {
   var script = document.createElement('script');
-  script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js';
+  script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js';
```

Comment:
I want to propose to add subresource-integrity verification for MathJax.
This would require us to stick to a specific version.
Since your rendering problems seem to be related to yesterdays MathJax update, this would probably be something we want anyways.

```suggestion
  script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3.1.0/es5/tex-chtml-full.js';
  // cryptographic hashes can be found here: 
  // https://www.jsdelivr.com/package/npm/mathjax?path=es5
  script.integrity = 'sha256-HJUiQvFxmEVWZ3D0qyz7Bg0JyJ2bkriI/WHQYo1ch5Y='
  script.crossOrigin = 'anonymous'
```